cmd/tendermint/commands/root_test.go

```diff
@@ -103,3 +104,55 @@ func TestRootConfig(t *testing.T) {
 	}
 
 }
+
+func TestParseInvalidConfig(t *testing.T) {
+	assert := assert.New(t)
+	cases := [...]struct {
```

Comment:
`[]` should work too

Reply:
Thanks for the suggestion @melekes, however in this case, when I get a failed test I'd like to easily jump to the index that failed, hence the `[...]` and subsequent `2: {}`

Reply:
I'll update this as agreed upon on in the meeting.